plugins/com.google.cloud.tools.eclipse.appengine.facets.test/src/com/google/cloud/tools/eclipse/appengine/facets/StandardFacetInstallDelegateTest.java

```diff
@@ -56,32 +56,36 @@ public void testCreateConfigFiles()
     delegate.createConfigFiles(project, AppEngineStandardFacet.JRE7, monitor);
 
     IFile appengineWebXml = project.getFile("src/main/webapp/WEB-INF/appengine-web.xml");
+    IFile loggingProperties = project.getFile("src/main/webapp/WEB-INF/logging.properties");
     Assert.assertTrue(appengineWebXml.exists());
-    
+    Assert.assertTrue(loggingProperties.exists());
```

Comment:
Might want to move the logging tests to a new method for clarity and test isolation and then test the contents and format of the file too.